app/assets/javascripts/tylium/modules/sidebar.js

```diff
@@ -15,10 +16,24 @@
       var that = this;
 
       $('[data-behavior~=sidebar-toggle]').on('click', function(e) {
+        if ($(this).is('[data-behavior~=nodes-tree]')) {
+          if (window.innerWidth < that.minBreakpoint) {
+            $(this).attr('data-behavior', 'nodes-tree sidebar-toggle');
+          } else {
+            $(this).attr('data-behavior', 'nodes-tree sidebar-toggle open-only');
+          }
+        }
+
```

Comment:
This is to allow the sidebar to be toggled by clicking the 'nodes' icon on mobile views (a trick to show the nodes menu in this new condensed format)